locales/en.json

```diff
@@ -1,99 +1,93 @@
+  "goals-data-text": "By gathering anonymous data that does not exist anywhere else we wish to map the world’s ideologies, how they shift over time, and how they react to global events. Data driven understanding of how our belief systems behave is invaluable knowledge, and may take an active role in shaping the political landscape for the better.",
+  "goals-reports-title": "Generate reports",
+  "goals-reports-text": "Data is meaningless when its raw information. In order to actualize our vision of providing a useful guide for global ideologies, we have to present this information in informative ways. Politivector will prepare reports of its most recent batches of anonymous data with data analysis and visualization to make a clearer picture of what the world believes.",
+  "plans-banner": "Take a look at our<br><span class=\"semi-bold\">Future Plans</span><br>for Politivector",
+  "plans-questions-title": "Questions",
+  "plans-questions-text": "Currently, the questions in the test are of our mildly informed making. These questions were formulated using the most basic topics of discussion in politics and political philosophy. Our intention, after Politivector proves itself to be a valuable project, is to rework the questions to make the evaluation algorithm more accurate and informational. <br><br>The ultimate goal regarding the veracity of the questions available in Politivector is to prepare a set of educated questions with experts and professors willing to participate in this project.",
+  "plans-data-title": "Data",
+  "plans-data-text": "Right now, Politivector obtains absolutely no data from its users whatsoever. While data collection and analysis for a small-scale project such as Politivector is a distant goal, we wish to be able to compare test takers’ results —anonymous results, of course— and present a comprehensive image regarding the distribution of ideologies across the world. <br><br>One thought-through feature is to add a country, and maybe city, selector. This location information provided would allow us to display ideological maps of our anonymous test takers’ countries. The information portrayed by such distribution and concentration maps could be especially useful in demographic, social, and political studies. <br><br>Another possibility is to apply various data science methods to the data provided to us by our test takers. One specific method is “clustering”. Clustering would allow us to group together data points, seeing if they agree on a common center. <br><br>This analysis could present many new perspectives regarding the status of political philosophy in the modern world, such as how fragmented certain ideologies are, which political parties best satiate their peoples’ needs, and where a newcomer into politics could align themselves as a starting point. <br><br>Data collection and analysis is probably the most intricate of our long-term goals, which is why it is the enhancement we wish to implement last.",
```

Comment:
( ͡° ͜ʖ ͡°)

Reply:
`  "features-anonymous-text": "Your test results and country are all we are going to get. No specific location, IP, or device.",
`